pkg/actor/decommission.go

```diff
@@ -58,7 +59,7 @@ func (d decommission) Handles(conds []api.ClusterCondition) bool {
 
 func (d decommission) Act(ctx context.Context, cluster *resource.Cluster) error {
 	log := d.log.WithValues("CrdbCluster", cluster.ObjectKey())
-	log.Info("check decommission oportunities")
+	log.V(int(zapcore.InfoLevel)).Info("check decommission oportunities")
```

Comment:
This seems like debug information, not info?

Rationale is that this log line is very similar to an "I AM HERE" message. It's only useful when trying to diagnose why the operator isn't doing what you expect it to be doing.